handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -248,6 +250,39 @@ protected void deallocate() {
         }
     }
 
+    /**
+     * Enables OCSP stapling on the {@link SSLEngine} and throws an {@link Exception} if OCSP stapling is not supported.
+     */
+    @UnstableApi
+    public void enableOcsp(ReferenceCountedOpenSslContext context) {
+        context.enableOcsp();
```

Comment:
Agree. Next PR fixes it. I've reduced it to:

```
SSLContext#enableOcsp
SSL#setOcspResponse
SSL#getOcspResponse
```

The `SSL#enableOcsp` call doesn't need to be exposed to the user.